src/spring/azext_spring/tests/latest/test_asa_app.py

```diff
@@ -322,6 +322,32 @@ def test_app_deploy_enterprise(self, file_mock):
         self.assertEqual({'applicationConfigurationService': {'configFilePatterns': 'my-pattern'}},
                          resource.properties.deployment_settings.addon_configs)
 
+    @mock.patch('azext_spring._deployment_uploadable_factory.FileUpload.upload_and_build')
```

Comment:
Please also add a  test that config `config_file_patterns` and check the `addon_configs`.